### PR TITLE
feat(engine) add heartbeat interval configuration

### PR DESCRIPTION
**Pull request by loiclau**
## Description

Since MON-6308, a new parameter has been added to Engine configuration.
We need to reflect it in the Centreon Web forms.

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [x] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 19.10.x
- [x] 20.04.x
- [x] 20.10.x
- [x] 21.04.x (master)

<h2> How this pull request can be tested ? </h2>

change option Configuration > Pollers > Engine configuration (admin option)
check in db table cfg_nagios

export configuration
check export file /var/cache/centreon/config/engine/

## Checklist

- [x] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

**Review by kduret — COMMENTED** [review]
Please update the title of the PR

File: lang/fr_FR.UTF-8/LC_MESSAGES/help.po [hunk 1]
```diff
@@ -3225,6 +3225,15 @@ msgstr ""
 "tous les hôtes ou services (par exemple ceux qui n'ont pas été spécifié "
 "explicitement par l'utilisateur)."
 
+#: centreon-web/www/include/configuration/configNagios/help.php:461
+msgid ""
+"Time interval in seconds between two heatbeat events. This event is the one responsible of the ‘Last Update’ "
```
Comment (cgagnaire) [inline lang/fr_FR.UTF-8/LC_MESSAGES/help.po:3230]:
```suggestion
"Time interval in seconds between two heartbeat events. This event is the one responsible of the ‘Last Update’ "
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/help.po [hunk 2]
```diff
@@ -3225,6 +3225,15 @@ msgstr ""
...
+"column update in the Pollers listing. Value must be between 5 and 600. Default value is 30."
+msgstr ""
+"Intervalle de temps en secondes entre deux événements heatbeat. Cet événement est le responsable de "
```
Comment (cgagnaire) [inline lang/fr_FR.UTF-8/LC_MESSAGES/help.po:3233]:
```suggestion
"Intervalle de temps en secondes entre deux événements heartbeat. Cet événement est le responsable de "
```

File: www/include/configuration/configNagios/help.php [hunk 3]
```diff
@@ -741,6 +741,11 @@
     "This option allows you to specify what kind of date/time format Monitoring Engine should use in the web " .
     "interface and date/time macros."
 );
+$help["instance_heartbeat_interval"] = dgettext(
+    "help",
+    "Time interval in seconds between two heatbeat events. This event is the one responsible of the ‘Last Update’ "
```
Comment (cgagnaire) [inline www/include/configuration/configNagios/help.php:746]:
```suggestion
    "Time interval in seconds between two heartbeat events. This event is the one responsible of the ‘Last Update’ "
```

**Comment by loiclau** [comment]
wait acceptance, export clapi should fail

**Review by jeremyjaouen — APPROVED** [review]

**Review by callapa — APPROVED** [review]

**Pull request merged**